lib/hash_ring/ets.ex

```diff
@@ -230,12 +232,16 @@ defmodule ExHashRing.HashRing.ETS do
     end
   end
 
+  defp do_find_nodes(_table, _gen, 0, _remaining, _hash, found, _found_length) do
```

Comment:
I'm not actually sure that this clause makes sense to have. Since `find_nodes/3` asserts that `num_nodes` is greater than 0, this will never be hit.

Should empty hash rings be allowed?

Reply:
Yeah, I just reran Dialyzer, and it's erroring on this line:

```
Compiling 1 file (.ex)
Checking PLT...
[:compiler, :elixir, :kernel, :stdlib]
PLT is up to date!
Starting Dialyzer
dialyzer args: [
  check_plt: false,
  init_plt: '/Users/quinn/repos/ex_hash_ring/_build/dev/dialyxir_erlang-21.3.8.9_elixir-1.8.2_deps-dev.plt',
  files_rec: ['/Users/quinn/repos/ex_hash_ring/_build/dev/lib/ex_hash_ring/ebin'],
  warnings: [:unknown]
]
done in 0m0.86s
lib/hash_ring/ets.ex:235: The pattern <__table@1, __gen@1, 0, __remaining@1, __hash@1, _found@1, __found_length@1> can never match the type <ets:tid(),integer(),pos_integer(),_,_,[any()],non_neg_integer()>
done (warnings were emitted)
```